lib/usd/utils/util.cpp

```diff
@@ -113,7 +113,7 @@ template <typename T>
 void replacePath(const UsdPrim& oldPrim, const SdfPath& newPath, const T& proxy, SdfListOpType op)
 {
     // set the listProxy based on the SdfListOpType
-    T::ListProxy listProxy = proxy.GetAppendedItems();
+    typename T::ListProxy listProxy{proxy.GetAppendedItems()}; 
```

Comment:
Make GCC compiler happy:

T::ListProxy is a dependent name and it needs to be followed by "typename" keyword to provide a hint to the compiler that an unknown identifier (ListProxy) is a type.  

See here:
https://stackoverflow.com/questions/610245/where-and-why-do-i-have-to-put-the-template-and-typename-keywords